CakePHP/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -150,7 +150,6 @@ protected function processReturn(File $phpcsFile, $stackPtr, $commentStart)
 
         $suggestedType = implode('|', $suggestedNames);
         if ($types !== $suggestedType) {
-            $error = 'Function return type "%s" is invalid';
```

Comment:
This might have been meant as concatinated message.

Reply:
But is not necessary with the detailed 2nd message in place :)